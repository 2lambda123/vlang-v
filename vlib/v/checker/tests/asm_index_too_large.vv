unsafe asm amd64 {
	mov eax, $1
	mov $0, eax
	: 
	: 
	: eax
}
