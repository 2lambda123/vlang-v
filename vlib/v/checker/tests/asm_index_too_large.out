vlib/v/checker/tests/asm_index_too_large.vv:2:12: error: index too large. largest index is -1, got 1
    1 | unsafe asm amd64 {
    2 |     mov eax, $1
      |               ^
    3 |     mov $0, eax
    4 |     :
vlib/v/checker/tests/asm_index_too_large.vv:3:7: error: index too large. largest index is -1, got 0
    1 | unsafe asm amd64 {
    2 |     mov eax, $1
    3 |     mov $0, eax
      |          ^
    4 |     : 
    5 |     :
